build.sh

```diff
@@ -15,10 +15,10 @@ esac
 done
 
 echo "Extracting configuration file..."
-if [ ! -f $HOME"/.Appliscale/cftool" ]; then
-    mkdir -p $HOME"/.Appliscale/cftool/"
+if [ ! -f ~/.config/cftool ]; then
```

Comment:
In _bash_ `$HOME` was actually a good idea - especially wrapped in double quotes.